README.md

```diff
@@ -21,6 +21,16 @@ and more.
 
 ## Installation
 
+<!-- tabs-open -->
+
+### Using Igniter (recommended)
```

Comment:
This is just the template we use for most of our own repos, I'm happy to make igniter a footnote or just remove it from the guide etc.